package.json

```
@@ -32,94 +33,86 @@
         ]
     },
     "scripts": {
-        "build": "cross-env NODE_ENV=production webpack --config webpack.config.js",
-        "start": "cross-env NODE_ENV=development webpack serve",
-        "start:https": "cross-env NODE_ENV=development webpack serve --https",
+        "build": "cross-env NODE_ENV=production vite build",
+        "dev": "cross-env NODE_ENV=development vite",
+        "preview": "cross-env NODE_ENV=production vite preview",
         "test": "cross-env NODE_ENV=test playwright test",
         "test:debug": "cross-env NODE_ENV=test playwright test --headed --project=chromium",
         "test:inspector-mode": "cross-env NODE_ENV=test PWDEBUG=1 playwright test --project=chromium",
-        "test:unit": "cross-env NODE_ENV=test vitest src"
+        "test:unit": "cross-env NODE_ENV=test vitest src",
+        "lint": "npx eslint src/"
```

Is `npx` needed here?

`eslint` is marked as a dev dependency and it should be directly accessible from package.json scripts no?

```suggestion
        "lint": "eslint src/"
```